### ISSUE TITLE
in Home Assistant, after in installation via HACS, APM integration does not show up in the integration tab. 

### ISSUE THREAD

**Issue by mahawana** [custom-components/alexa_media_player#1393]
in my Home Assistant instance, running on a virtual machine, I installed this AMP integration via HACS.  However, once I rebooted the Home Assistant this APM integration does not show up in the list of integrations...

Error signature:  NONE...  Also, I looked in "Home-assistant.log" file, but no useful or error message related to Alexa or otherwise. 

system info: 
Alexa Media Player:  version  v3.10.10
HACS version:  1.15.2
Home Assistant verion:  2021.10.6
Home Assistant is running on Oracle VM Virtual box:  version  6.1.26

**Comment by OJBender** [comment]
Have you clicked on the lower right to ADD the integration? It does not show up in integrations automatically.

**Comment by yadavvineet** [comment]
i have installed the AMP for long time. since last update, its now showing in there. 
![image](https://user-images.githubusercontent.com/7949851/139736535-d8f2577f-fe15-4b0f-a531-3dee67342363.png)

Also, i am not able to re-authenticate to amazon, earlier i just put in opt and login, now it just not accepts saying invalid password. Everything is correct though

**Comment by mahawana** [comment]
Hi OJBender, thank you very much for the feedback.  
Yes, before posting my request for help, I had  pushed  the "+add integration" button few times but I could not see it. I spent an hour or so (including a restart of HA), but could not see this integration. 

However, today as I was writing the email response, I was looking for some details, and I checked one more time and this time, I saw the configuration. 
it seems that there is a significant delay between installing the configuration and then seeing it in the configuration menu. I really don't know why...is it supposed to take a long time ? perhaps I was simply impatient ? 
 
Anyway, again thanks for the help.  I'm now  able to go to the next step to install AMP.
